scraper/README.md

```diff
@@ -16,20 +16,11 @@ docker-compose -f <FILE> build && docker-compose -f <FILE> up
 We have both a development and a production version pre-configured.
 
```

Comment:
Do you think the other possibilities of configuration could be explained here as well? Even though they are not to be sent through the 'docker-compose.yml' file they are still configurable, right?

Reply:
You are right, that is a good idea. Here I could explain more about the scraper, specifically. I'll add that in the next PR.